eng/readme-templates/Url.md

```diff
@@ -8,7 +8,7 @@
 when(ARGS["readme-host"] = "mar",
     when(ARGS["is-product-family"],
         cat("https://mcr.microsoft.com/catalog?search=", ARGS["repo"], "/"),
-        cat("https://mcr.microsoft.com/product/", ARGS["repo"])),
+        cat("https://mcr.microsoft.com/en-us/product/", ARGS["repo"], "/about/")),
```

Comment:
We should avoid linking to culture-specific URLs. In this case, we can reference `https://mcr.microsoft.com/product` instead. This will require regenerating the readmes.

```suggestion
        cat("https://mcr.microsoft.com/product/", ARGS["repo"], "/about/")),
```